lib/core/src/lib/card-view/components/card-view-keyvaluepairsitem/card-view-keyvaluepairsitem.component.html

```diff
@@ -23,44 +20,43 @@
         </mat-table>
     </div>
 
-
-    <div class="adf-card-view__key-value-pairs adf-property-value"
-         *ngIf="isEditable() && values && values.length">
+    <div class="adf-card-view__key-value-pairs adf-property-value" *ngIf="isEditable() && values && values.length">
         <div class="adf-card-view__key-value-pairs__row">
-            <div class="adf-card-view__key-value-pairs__col">{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.NAME' | translate }}</div>
-            <div class="adf-card-view__key-value-pairs__col">{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.VALUE' | translate }}</div>
+            <div class="adf-property-col-key">{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.NAME' | translate }}</div>
+            <div class="adf-property-col-value">{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.VALUE' | translate }}</div>
         </div>
 
-        <div class="adf-card-view__key-value-pairs__row"
-             *ngFor="let item of values; let i = index">
-            <div class="adf-card-view__key-value-pairs__col">
-                <mat-form-field>
-                    <input matInput
-                           placeholder="{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.NAME' | translate }}"
-                           (blur)="onBlur(item.value)"
-                           [attr.data-automation-id]="'card-'+ property.key +'-name-input-' + i"
-                           [(ngModel)]="values[i].name">
-                </mat-form-field>
+        <div class="adf-card-view__key-value-pairs__row" *ngFor="let item of values; let i = index">
+            <div class="adf-property-col-key">
```

Comment:
Can we apply this class to input directly and remove div?

Reply:
Done